pyinfra/api/inventories/kubernetes.py

```diff
@@ -0,0 +1,26 @@
+from pyinfra.api.connectors.util import run_local_process, split_combined_output
```

Comment:
I think this file/function should live within the `pyinfra.api.connectors` module. Could either be inside `api/connectors/kubernetes.py` or make `api/connectors/kubernetes/__init__.py` + `api/connectors/kubernetes/util.py`.